apps/src/levelbuilder/lesson-editor/UploadImageDialog.jsx

```diff
@@ -78,13 +80,22 @@ export default function UploadImageDialog({
     handleDialogClose();
   };
 
+  const dimensionsText = dimensions
+    ? `Max dimensions: ${dimensions.width} x ${dimensions.height}`
+    : '';
+  const fileTypesText = fileTypes
+    ? `Supported file types: ${fileTypes.join(', ')}`
+    : '';
   return (
     <LessonEditorDialog isOpen={isOpen} handleClose={handleDialogClose}>
       <h2>Upload Image</h2>
       {
         // TODO: A11y279 (https://codedotorg.atlassian.net/browse/A11Y-279)
         // Verify or update this alt-text as necessary
       }
+      {dimensionsText && <p>{dimensionsText}</p>}
+      {fileTypesText && <p>{fileTypesText}</p>}
+      {<p>Images must be no larger than 2MB</p>}
```

Comment:
Nit: Could use a const for max file size and then use consistent display text as in Bryan's suggestion.
```
`Max file size: ${MAX_FILE_SIZE} MB`
```

Reply:
Updated!